plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -73,26 +71,27 @@ func TestWinPerfcountersConfigGet2(t *testing.T) {
 
 	perfobjects[0] = PerfObject
 
-	m := Win_PerfCounters{PrintValid: false, TestName: "ConfigGet2", Object: perfobjects}
+	m := Win_PerfCounters{PrintValid: false, Object: perfobjects}
 
-	err := m.ParseConfig(&metrics)
+	err := m.ParseConfig()
 	require.NoError(t, err)
 
-	if len(metrics.items) == 1 {
+	var parsedItems = m.GetParsedItemsForTesting()
```

Comment:
Added GetParsedItemsForTesting to satisfy existing tests without too drastic a rewrite